README.md

```diff
@@ -1,17 +1,17 @@
 # What is NeuroLibre?
 
-Neurolibre is a curated repository of interactive neuroscience notebooks, seamlessly integrating data, text, code and figures. Notebooks can be freely modified and re-executed through the web, offering a fully reproducible, “libre” path from data to figures. Neurolibre is powered by the Binder project, with computational resources provided by [CONP](http://conp.ca/), [CBRAIN](http://mcin.ca/technology/cbrain/) and [Compute Canada]().
+NeuroLibre is a curated repository of interactive neuroscience notebooks, seamlessly integrating data, text, code and figures. Notebooks can be freely modified and re-executed through the web, offering a fully reproducible, “libre” path from data to figures. NeuroLibre is powered by the Binder project with computational resources provided by [CONP](http://conp.ca/), [CBRAIN](http://mcin.ca/technology/cbrain/), and [Compute Canada]().
 
 # What can be published on NeuroLibre?
 
 NeuroLibre currently welcomes submissions along two tracks:
  - **Tutorial** notebooks related to a workshop in the neuroscience field.
  - **Companion** notebooks for articles posted as preprints in the neuroscience field.
-These workshops Notebooks are typically expected to use less than 100 GB, and run in less of an hour, although exceptions may be granted on a case by case basis. 
+These workshops notebooks are typically expected to use less than 100 GB, and run in less of an hour, although exceptions may be granted on a case by case basis. 
 
 # How are NeuroLibre submissions reviewed?
 
-They are not, in the sense of a traditional peer-reviewed publication. The NeuroLibre team does check that the notebooks run correctly, and also checks how much compute time and data storage are required. They also check that the submission fits one of the two tracks above. NeuroLibre commits to provide computational resources for free, yet this investment needs to benefit the Open NeuroScience community. The Neurolibre team reserves the right of rejecting a submission if the content of the notebook is judged to fall outside Neuroscience, requires excessive resources for the proposed scope of work, or is of obvious unsufficient quality.
```

Comment:
actually for this part we have to agree. Are we doing a scientific peer review, leave it to another organization or not do it ? I feel it is important to do a minimum science peer review so people don't push random things on neurolibre ?
@pbellec @emdupre

Reply:
I have made a few changes to that paragraph to reflect the conclusions of the HBM hackathon round table. Basically, neurolibre provides a **technical** review, distinct and complementary to traditional scientific review.